source/utilities.tex

```diff
@@ -18990,7 +18990,7 @@
 \begin{example}
 \begin{codeblock}
 using namespace std;
-vector<int> v = ...
+vector<int> v = ...;
```

Comment:
Thanks, I didn't know about that. There seem to be many uses of literal `...` that could be changed, but I'll leave the others for a different PR.